src/handlers/yarn.js

```diff
@@ -0,0 +1,51 @@
+import ora from 'ora';
+import fetch from 'isomorphic-fetch';
+import { exec, log } from '../util';
+
+let originalVersion;
+
+async function afterAll() {
+  await exec(`yarn add ${originalVersion} --no-lockfile`);
```

Comment:
Unfortunately Yarn doesn't allow you to update a dependency without updating package.json so we have to put things back the way they were at the end.